features/checkout/paying_for_order/changing_offline_payment_method_after_order_confirmation.feature

```diff
@@ -33,6 +33,40 @@ Feature: Changing the offline payment method after order confirmation
         And I retry the payment with "Offline" payment method
         Then I should have chosen "Offline" payment method
 
+    @ui
```

Comment:
Cover these scenarios in API context or add `@no-api` tag if it is not possible

Reply:
Did what I could, the rest went to `@no-api` as the concept of paying with an offline payment method is either nonexistent or not covered at all in the API.